tools/mage/deploy/deploy.go

```diff
@@ -591,3 +586,32 @@ func customResourceVersion() string {
 	// and also the same format used by the master stack.
 	return fmt.Sprintf("%s (%s)", util.Semver(), util.CommitSha())
 }
+
+// Takes a string  and returns the panther- prefixed, lowercased slice of words(strings) (separated by spaces).
+// e.g "oRg-ApI" -> []string{"panther-org-api"}
+// e.g "one two THREE" -> []string{"panther-one", "panther-two", "panther-three"}
+func PantherNames(setString string) []string {
+	set := strings.Fields(setString)
+	for i, entry := range set {
+		entry = strings.ToLower(entry)
+		if !strings.HasPrefix(entry, "panther-") {
+			entry = "panther-" + entry
+		}
+		set[i] = entry
+	}
+	return set
+}
+
+// Call a method for every string in the callOnSet string slice. Return a slice of errors where the
+// index of the error is the index of the callOnSet string used as the argument in the function call.
+func CallForEachString(label string, callOnSet []string, callFn func(string) error) (callErrors []error) {
+	for _, setEntry := range callOnSet {
+		log.Infof("%s: %v", label, setEntry)
+		err := callFn(setEntry)
+		if err != nil {
+			log.Errorf("%s: %s %v", label, setEntry, err)
+		}
```

Comment:
Nit: you could put the error on the same line

```suggestion
		if err := callFn(setEntry); err != nil {
			log.Errorf("%s: %s %v", label, setEntry, err)
		}
```

Reply:
I would but I want to capture nil errors for counting success / error rates and have the returned error set indexable by the callOnSet items. If the caller needs to know which argument resulted in the error they can use something like `callset[indexOf(error)]`

Reply:
Oh right, I forgot you're using that variable later. Carry on